Library/List/SubscriptionAddOnList.cs

```diff
@@ -6,6 +6,8 @@ public class SubscriptionAddOnList : RecurlyList<SubscriptionAddOn>
     {
         private Subscription _subscription;
 
+        public SubscriptionAddOnList() {}
```

Comment:
It seems pointless to require a subscription just to check the currency. It made sense with the old pattern, but now with the SubscriptionChange object, you don't always have a Subscription object, and you shouldn't need one. Besides, client side validation is bad.